### ISSUE TITLE
The visual presentation of the Components/Images don't have a contrast ratio of at least 3:1 against adjacent color, in Admin screen.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#853]
**Describe the bug**
The visual presentation of the Components/Images don't have a contrast ratio of at least 3:1 against adjacent color.

**To Reproduce**

1. Install "Color Contrast Analyzer" extension in your browser.
2. Go to Admin  (http://localhost:9992/cm/app/?view=workflow&site=test&path=%2FSites%2Ftest%2F)
3. Scroll to the upper part of the page.
4. Click "Extension" icon given in the chrome browser, in top right. It will open Extension Pop Up.
5. Click "Color Contrast Analyzer", option in pop up. It will open another pop up.
6. Click "Capture Visible Content" from the pop up.
7. Observe that Arrows icons given between workflows sections. They are not highlighted (black and white) by the Color Contrast Analyzer.

**Expected behavior**
The visual presentation of the Components/Images should have a contrast ratio of at least 3:1 against adjacent color.

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by natechadwick** [comment]
@ramdev1980  Use Gimp to recolor those images with darker backgrounds.  For the light blue - replace with the dark blue used elsewhere in the UI.
